Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.cs

```diff
@@ -186,6 +197,16 @@ protected override void OnApplyTemplate()
                 ArrangeForTouch();
             }
 
+            UpdateMinimumDisplayText(Minimum, this);
+            UpdateMaximumDisplayText(Maximum, this);
+
+            //  Measure our min/max text longest value so we can avoid the length of the scrolling reason shifting in size during use.
+            var tb = new TextBlock { Text = Maximum.ToString() };
+            tb.Measure(new Size(Double.PositiveInfinity, Double.PositiveInfinity));
+
+            _minValueText.MinWidth = tb.ActualWidth;
```

Comment:
If someone is using the old template, this might crash - make sure these changes work with the old template

Reply:
I added some null checks around the tooltip and min/max value textblocks code. This allows the old template to continue to work with the new logic. 👍